### DISCUSSION TITLE
Wemos mini ESP8266 / MAX7219 segment led

### DISCUSSION THREAD

**Discussion by Unco16** [noah1510/LedController#24] [Q&A] [answered]
Hi, 
Thanks for the project!
I struggled a bit to the LCDemo7Segment example with ESP8266 (Wemos d1 mini) but finally got it working.
If it can help people in the future, here is the pin that I use and settings that I have changed into the code to match the pin.
Wemos                -->  MAX7519 7 Segments display
Pin D8 (GPIO 15) --> DIN
Pin D7 (GPIO 13) --> CS
Pin D5 (GPIO 14) --> CLK

In the code, I have changed the following line:
lc=LedController<1,1>(12,11,10);
To:
lc=LedController<1,1>(13,14,15);

**Comment by noah1510** [comment, marked as answer]
Thanks for the feedback, I think I will make your proposed changes the default.
This change might also be required for an ESP32.

Since that example is simply one of the examples from wayoda/LedControl, I only changed the parts that where required to get it to compile but could not test them on real hardware.
I will also define DIN,CS and CLK as variables so that they are better to tweak when necessary.

**Reply by noah1510** [reply]
c297453150ddde2ffe1de9c0282cd26400eab57f has the mentioned changes.
The notice that the pins might not work with your board are now in a comment just above them.
